source/agora/test/ManyValidators.d

```diff
@@ -51,7 +50,8 @@ unittest
     network.waitForDiscovery();
 
     auto nodes = network.clients;
-    network.expectBlock(Height(7));
+    Height expected_block = Height(conf.extra_blocks);
+    network.expectBlock(expected_block++);
```

Comment:
I would really prefer not using postfix here, and moving this outside (to make it clear).

```suggestion
    network.expectBlock(expected_block);
    expected_block++;
```

Reply:
As we agreed verbally, I will keep the postfix operator.

Reply:
And I will drink more coffee.